unit-tests/run-unit-tests.py

```diff
@@ -28,18 +28,21 @@ def usage():
     print( '        --debug          Turn on debugging information' )
     print( '        -v, --verbose    Errors will dump the log to stdout' )
     print( '        -q, --quiet      Suppress output; rely on exit status (0=no failures)' )
-    print( '        -r, --regex      run all tests that fit the following regular expression' )
-    print( '        -s, --stdout     do not redirect stdout to logs' )
-    print( '        -t, --tag        run all tests with the following tag. If used multiple times runs all tests matching' )
+    print( '        -r, --regex      Run all tests that fit the following regular expression' )
+    print( '        -s, --stdout     Do not redirect stdout to logs' )
+    print( '        -t, --tag        Run all tests with the following tag. If used multiple times runs all tests matching' )
     print( '                         all tags. e.g. -t tag1 -t tag2 will run tests who have both tag1 and tag2' )
     print( '                         tests automatically get tagged with \'exe\' or \'py\' and based on their location' )
     print( '                         inside unit-tests/, e.g. unit-tests/func/test-hdr.py gets [func, py]' )
-    print( '        --list-tags      print out all available tags. This option will not run any tests' )
-    print( '        --list-tests     print out all available tests. This option will not run any tests' )
+    print( '        --list-tags      Print out all available tags. This option will not run any tests' )
+    print( '        --list-tests     Print out all available tests. This option will not run any tests' )
     print( '                         if both list-tags and list-tests are specified each test will be printed along' )
     print( '                         with what tags it has' )
-    print( '        --no-exceptions  do not load the LibCI/exceptions.specs file' )
-    print( '        --context        The context to use for test configuration' )
+    print( '        --no-exceptions  Do not load the LibCI/exceptions.specs file' )
+    print( '        --context <>     The context to use for test configuration' )
+    print( '        --repeat <#>     Repeat each test <#> times' )
+    print( '        --config <>      Ignore test configurations; use the one provided' )
+    print( '        --no-reset       Do not try to reset any devices, with or without Acroname' )

```

Comment:
Where is the `no-reset` option used? Looks to me like it's never referenced after becoming `true`

Reply:
Line 401